src/main/resources/liquibase/DDP-5672-longitudinal-wait.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
can we stay with the same naming convention for the updateDB xmls please

Reply:
Yes...if you tell me what that is :)

Reply:
045-updateDB.xml should be that one